sdk/tables/Azure.Data.Tables/src/TableServiceClient.cs

```diff
@@ -84,6 +100,46 @@ public TableServiceClient(Uri endpoint, TableSharedKeyCredential credential, Tab
             Argument.AssertNotNull(credential, nameof(credential));
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TableServiceClient"/>.
+        /// </summary>
+        /// <param name="connectionString">
+        /// A connection string includes the authentication information
+        /// required for your application to access data in an Azure Storage
+        /// account at runtime.
+        ///
+        /// For more information,
+        /// <see href="https://docs.microsoft.com/azure/storage/common/storage-configure-connection-string">
+        /// Configure Azure Storage connection strings</see>.
+        /// </param>
+        /// <param name="options">
+        /// Optional client options that define the transport pipeline policies for authentication, retries, etc., that are applied to every request.
+        /// </param>
+        public TableServiceClient(string connectionString, TableClientOptions options = null)
```

Comment:
Should we call the internal ctor from this one?

Reply:
I wanted to do that but the arguments all rely on the result of `TableConnectionString.Parse`